### DISCUSSION TITLE
Svelte official support

### DISCUSSION THREAD

**Discussion by techgems** [tailwindlabs/headlessui#435] [Ideas]
So this one is kinda self explanatory. The work for Headless UI thus far is amazing! I can't emphazise that enough, this request is to bring some of that wonderful work into Svelte as well, other than just Vue and React.

I am willing to collaborate on this too in porting react equivalents into Svelte ones.

**Comment by AndrewEQ** [comment]
Svelte FTW!

**Comment by grandslammer** [comment]
+1

**Comment by ZGuardian** [comment]
You can use/contribute to this one here – [svelte headless ui](https://github.com/rgossiaux/svelte-headlessui).

**Comment by sebastianconcept** [comment]
+1 Svelte is a must!

**Comment by ashxdev** [comment]
+1 Svelte
